src/api/Feed.js

```diff
@@ -286,7 +293,7 @@ class Feed extends Base {
             if (!this.isDestroyed()) {
                 this.setCachedItems(id, sortedFeedItems);
                 if (this.hasError) {
-                    errorCallback(sortedFeedItems);
+                    errorCallback(sortedFeedItems, this.errors);
```

Comment:
Should we update each constituent promise to bind its `reject` callback to `fetchFeedItemErrorCallback` along with `resolve`? Each error callback could then pass its error object to `reject`. That would allow us to use `.catch`  on the main Promise.all promise chain above, which would remove the need for an instance level `errors` property.

Reply:
`Promise.all` will reject immediately upon any of the input promises rejecting, which is not the current behavior.